jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java

```diff
@@ -1,80 +0,0 @@
-package io.github.jhipster.config.metric;
```

Comment:
Why is this class removed !?

Reply:
Probably a bad merged from master to the sb branch

Reply:
I added back files